src/theme/components/wishlist/Wishlist.vue

```diff
@@ -0,0 +1,100 @@
+<template>
```

Comment:
Layout and display of items and function similar to cart, but we need to handle wording, functions and events individually. Is there a way to keep repeated code as less as possible?